src/sundials/sundials_context.c

```diff
@@ -30,30 +30,34 @@
 void sunAdiakCollectMetadata();
 #endif
 
-int SUNContext_Create(void* comm, SUNContext* sunctx)
+int SUNContext_Create(void* commptr, SUNContext* sunctx)
```

Comment:
We can change this to take a `SUN_Comm` but this will definitely force every user to need to make a change since every user calls `SUNContext_Create`. I have done the work (it was a simple find and replace mostly) if we want to go that route (its on a different branch I have not pushed up). @gardner48 What are your thoughts?

Reply:
I think we should be consistent about using the new communicator type and not have a mix of `void*` communicators and `SUNComm` communicators.

Reply:
Done